src/sinks/console.rs

```diff
@@ -142,7 +142,11 @@ impl StreamSink for WriterSink {
                     // so stop the sink.
                     error!("Error writing to output: {}. Stopping sink.", error);
                     return Err(());
-                }
+                };
```

Comment:
I don't think this `;` is necessary.

Reply:
Actually, I think I'm wrong :smile:

Reply:
I'm pretty sure you need to get rid of it :smile: